pyproject.toml

```
@@ -64,7 +64,7 @@ dependencies = [
     "tabulate",
     "torch",
     "tqdm",
-    "transformers>=4.51.3", # attention api + qwen
+    "transformers>=4.53.0", # attention api + qwen
```

```suggestion
    "transformers>=4.53.0", # update_causal_mask
```

the comment gives us a hint to why we asked for this particular min version.